### PR TITLE
Feature/searchresults 40285870

### PR DESCRIPTION
**Pull request by ihor-stack**
2.2 Search Result

### CODE REVIEW

File: src/common/components/Header/Header.tsx [hunk 1]
```diff
@@ -54,8 +65,19 @@ const SearchBar = () => {
...
         />
       </div>
+
+      <SearchResults
+        open={openSearchResults}
+        onClose={toggleSearchResults}
+        foundNFTs={mockNFTs}
+        foundMembers={mockMembers}
+        onChangeTimeRange={() => {
+          /**/
```
Comment (ngvtuan) [inline src/common/components/Header/Header.tsx:78]: Can we remove this line?

File: src/features/searchResults/mockData.ts [hunk 2]
```diff
@@ -0,0 +1,78 @@
+import { TMemberCard } from 'features/members/MemberCard'
+import { TNFTCard } from '../../common/components/NFTCard'
+import mockAvatar from '../../common/assets/images/avatar2-placeholder.png'
+import image from '../../common/assets/images/nft-card-placeholder.png'
```
Comment (ngvtuan) [inline src/features/searchResults/mockData.ts:4]: Can we change the variable name to `nftCard`?

**Review by ngvtuan — COMMENTED** [review]
Issue 1
Please reduce the width of the pop up

Issue 2
1. Reduce the padding of 2 items
2. Can we add 3 rows like the design?

**Review by CrazyOptimist — APPROVED** [review]

File: src/features/searchResults/mockData.ts (hunk 2 above)
Comment (ihor-stack) [reply src/features/searchResults/mockData.ts:4]: We discussed as "T". Need to change again?

Comment (ngvtuan) [reply src/features/searchResults/mockData.ts:4]: @ihor-stack I think we just need change the name from `image` to `nftCard` is ok.

File: src/common/components/Header/Header.tsx (hunk 1 above)
Comment (ihor-stack) [reply src/common/components/Header/Header.tsx:78]: If I remove comment line, eslint will warn about empty func body.

**Review by ngvtuan — APPROVED** [review]

**Pull request merged**